apps/storybook/stories/components/search/search-async/SearchAsync.mdx

```diff
@@ -1,33 +1,152 @@
+import { Text, Code, Box, Link, UnorderedList, ListItem } from "@kvib/react/src/";
 
 <Meta of={SearchAsyncStories} />
 
 # SearchAsync
 
 SearchAsync er et søkefelt som laster inn søkeresultater fra en liste med objekter. Resultatene blir vist i en dropdown der brukeren kan velge hvilket resultat som skal bli valgt.
 
-## Ta i bruk
-
 ```jsx
 import { SearchAsync } from "@kvib/react";
 ```
 
-## Props
-
-<Canvas of={SearchAsyncStories.SearchAsync} />
-<Controls of={SearchAsyncStories.SearchAsync} />
-
-## Async søkeresultater
+<DocsContainer>
+
+**Egnet til:**
+
+- Raskt finne relevant innhold
+- Laste inn innhold asynkront
+- Select med søk
+
+**Uegnet til:**
+
+- Innhold som lett kan navigeres til uten søk
+
+<Feedback component="SearchAsync" />
+
+</DocsContainer>
+
+<DocsHeading>Alternativer</DocsHeading>
+<DocsContainer>
+
+<DocsStory
+  title="Required props"
+  description={
+    <Box>
+  <Text as="b">loadOptions</Text>
+  <Text>SearchAsync komponentens prop <Code>loadOptions</Code> er en funksjon ment for asynkron lasting av alternativer basert på brukerinput i inputfeltet. Den tar to parametere:</Text>
+  <UnorderedList>
+    <ListItem>
+      <Text><Code>inputValue</Code>: Teksten brukeren skriver inn.</Text>
+    </ListItem>
+    <ListItem>
+      <Text>
+        <Code>callback</Code>: En funksjon som tar en liste av objekter (<Code>options</Code>) og brukes til å returnere søkeresultatene. Hvert objekt i <Code>options</Code> må ha en <Code>label</Code> (tekst vist i dropdown) og en <Code>value</Code> (verdien returnert ved valg).
+      </Text>
+    </ListItem>
+  </UnorderedList>
+
+  <Text as="b">onChange</Text>
+  <Text><Code>onChange</Code> er en funksjon som håndterer endringer i valgt verdi. Den tar ett parameter:</Text>
+  <UnorderedList>
+    <ListItem>
+      <Text>
+        <Code>newValue</Code>: Den nye verdien som er valgt av brukeren. Denne verdien er av typen <Code>T</Code> eller <Code>null</Code>. <Code>T</Code> representerer typen av objektene i dine <Code>options</Code>, og <Code>null</Code> brukes når det ikke er noen valgt verdi.
+      </Text>
+    </ListItem>
+  </UnorderedList>
+</Box>
+}
+story={<Canvas of={SearchAsyncStories.SearchAsyncResults} />}
+/>
 
-<Canvas of={SearchAsyncStories.SearchAsyncResults} />
+<DocsStory
+  title="Utseende"
+  description={
+    <Box>
+      <Text>
+        SearchAsync har fire varianter. <Code>outline</Code> er standardvarianten og skal brukes i de fleste tilfeller.
+        <Code>filled</Code> kan brukes som et alternativ til outline. For mindre viktige søkefelt kan
+        <Code>flushed</Code> og
+        <Code>unstyled</Code> brukes.
+      </Text>
+      <Text>
+        Det er mulig å velge mellom tre forskjellige størrelser: <Code>sm</Code>, <Code>md</Code> og <Code>lg</Code>.
+        <Code>md</Code> er standard.
+      </Text>
+      <Text>
+        For å endre borderColor når søkefeltet er i fokus kan du sette farge med color-token (f.eks. green.500) i{" "}
```

Comment:
Hva er default-farge for focusBorderColor?  Siden ikke vi har kodet noe for det selv, så gjetter jeg at det er blå eller grønn avhengig av valgt farge for prosjektet. Det kan kanskje legges til en setning om det hvis det er tilfelle. I så fall er det jo veldig sjelden at det vil være behov for å endre fargen.